src/traces/treemap/attributes.js

```diff
@@ -0,0 +1,274 @@
+/**
+* Copyright 2012-2019, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var hovertemplateAttrs = require('../../plots/template_attributes').hovertemplateAttrs;
+var texttemplateAttrs = require('../../plots/template_attributes').texttemplateAttrs;
+
+var colorScaleAttrs = require('../../components/colorscale/attributes');
+var domainAttrs = require('../../plots/domain').attributes;
+var pieAttrs = require('../pie/attributes');
+var sunburstAttrs = require('../sunburst/attributes');
+var constants = require('./constants');
+var extendFlat = require('../../lib/extend').extendFlat;
+
+module.exports = {
+    labels: sunburstAttrs.labels,
+    parents: sunburstAttrs.parents,
+
+    values: sunburstAttrs.values,
+    branchvalues: sunburstAttrs.branchvalues,
+    count: sunburstAttrs.count,
+
+    level: sunburstAttrs.level,
+    maxdepth: sunburstAttrs.maxdepth,
+
+    tiling: {
```

Comment:
What about using `tile` instead of `tiling`?

Reply:
I prefer tiling/packing to tile/pack